### ISSUE TITLE
MoralisInterface.LogInAsync(user, pass) Does Not Set Current User

### ISSUE THREAD

**Issue by vrxgaminginc** [ethereum-boilerplate/web3-unity-boilerplate#95]
Steps
1. Create a new project with moralisweb3sdk_v1_1_0.unitypackage
2. Add appid and server settings, then append this code to MoralisController.cs:116
```
                if (MoralisInterface.IsLoggedIn())
                    await MoralisInterface.LogOutAsync();

                MoralisUser user = await MoralisInterface.LogInAsync("user", "pass");
                Debug.Log($"TEST LogInAsync() {user.objectId} ");    // valid                
                Debug.Log($"TEST LoggedIn? {MoralisInterface.IsLoggedIn()} ");    // false, not logged in 
```
Expected Result: User is logged in after logging in successfully with LogInAsync()
Actual Result: User is not logged in even though LogInAsync was successful

**Comment by xactant** [comment]
Fix for this will be included in the release this week (May 19, 2022).

**Comment by KoenRijpstra** [comment]
Fixed in the new 1.2.0 release 🥳
